gammapy/data/tests/test_data_store.py

```diff
@@ -150,3 +151,40 @@ def test_make_psf(pars, result):
     assert_quantity_allclose(psf.energy[10], result["psf_energy"])
     assert_quantity_allclose(psf.exposure[10], result["psf_exposure"])
     assert_quantity_allclose(psf.psf_value[10, 50], result["psf_value"])
+
+
+@requires_dependency('scipy')
+@requires_data('gammapy-extra')
+def test_make_meanedisp(tmpdir):
+    position = SkyCoord(83.63, 22.01, unit='deg')
+    store = gammapy_extra.filename("datasets/hess-crab4-hd-hap-prod2")
+    data_store = DataStore.from_dir(store)
+
+    obs1 = data_store.obs(23523)
+    obs2 = data_store.obs(23592)
+    obslist = ObservationList([obs1, obs2])
+
+    e_true = EnergyBounds.equal_log_spacing(0.01, 150, 80, "TeV")
+    e_reco = EnergyBounds.equal_log_spacing(0.5, 100, 15, "TeV")
+    rmf = obslist.make_mean_edisp(position=position, e_true=e_true, e_reco=e_reco)
+
+    assert len(rmf.e_true.nodes) == 80
+    assert len(rmf.e_reco.nodes) == 15
+    assert_quantity_allclose(rmf.data[53, 8], 0.0559785805550798)
+
+    rmf2 = obslist.make_mean_edisp(position=position, e_true=e_true, e_reco=e_reco,
+                                   low_reco_threshold=Energy(1, "TeV"), high_reco_threshold=Energy(60, "TeV"))
+    # Test that the energy dispersion is equal to zero for the reco energy bin inferior to low_reco_threshold and
+    #  superior to high_reco_threshold
+    i2 = np.where(rmf2.data[:, 13] != 0)[0]
```

Comment:
It does show up here:
http://docs.gammapy.org/en/latest/api/gammapy.utils.nddata.NDDataArray.html#gammapy.utils.nddata.NDDataArray.evaluate

This can be configured globally or per-class in Sphinx, whether inherited attributes are shown for sub-classes. At the moment the Gammapy (and I think Astropy) docs don't show inherited methods, you have to click on the link to the base class at the top.

I'm not sure if changing the default to always show inherited attributes / methods, or to do it on a case-by-case basis to get "optimal" API docs is an improvement / maintainable.

@joleroi or @JouvinLea - If you think this could / should be improved, please open a separate issue stating your preference.